documentation/source/quickstart.rst

```diff
@@ -302,41 +302,41 @@ We can also cast the signal handle directly to an integer:
 
 
 
-Parallel and sequential execution
+Blocking and concurrent execution
```

Comment:
Users searching for "parallel" or "sequential" should still be able to find this section, can we offer these terms in parentheses maybe?

Reply:
Is there someway to make a search for those terms drop them at this section without displaying the terms? "Parallel" is not correct.

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-only
seems to suggest that ``.. only:: never`` (assuming ``never`` being undefined and thus ``False``) could work.

Reply:
While that successfully generated nothing, it does not turn up in a search.

Reply:
Ah, too bad.

While something could probably be done with https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_object_type, it's not worth the effort.

Reply:
I'll change the heading back for now.